### DISCUSSION TITLE
Align button to form inputs

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1224] [Elements (button, image, label, loader, ...)] [answered]
Hello,
is there any Fomantic-UI class to have buttons aligned to inputs within a form?

I mean something like this:

![form](https://user-images.githubusercontent.com/5399265/70892083-c3701e00-1fe8-11ea-832a-047472c79856.png)

Here is my [Plunker](https://plnkr.co/edit/r0CsKZpRTNMeBk7xHAl2?p=preview).

**Comment by prudho** [comment, marked as answer]
Actually no there's no class to do this, but there's a little trick that'll do the thing:
```html
<div class="field">
  <label>&nbsp;</label>
  <div class="ui submit button">Submit</div>
</div>
```

You can add a label containing no text so the button will be aligned with inputs: https://next.plnkr.co/edit/Uv4nFvSXmUQwJJtf

**Comment by smartm0use** [comment]
Yes, it is exactly what I did but I do not like it. Isn't it a dirty trick?

**Comment by smartm0use** [comment]
What about to find a better way?

**Comment by smartm0use** [comment]
Finally solved with this:

```
<div class="ui input">
  <input type="text" placeholder="Search...">
  <button class="ui icon button">
    <i class="plus icon"></i>
  </button>
</div>
```

```
button {
  margin-left: 10px !important;
}
```

https://jsfiddle.net/mt78nx2o/

**Comment by lubber-de** [comment]
Closing because solutions are given/found using standard FUI. @fomantic/admins  If you think, we have to implement some extra class for that, then reopen the issue
